osgp/protocol-adapter-dlms/osgp-protocol-adapter-dlms/src/main/java/org/opensmartgridplatform/adapter/protocol/dlms/domain/commands/configuration/service/ProtocolServiceLookup.java

```diff
@@ -0,0 +1,31 @@
+package org.opensmartgridplatform.adapter.protocol.dlms.domain.commands.configuration.service;
+
+import java.util.List;
+
+import org.opensmartgridplatform.adapter.protocol.dlms.domain.entities.Protocol;
+import org.opensmartgridplatform.adapter.protocol.dlms.exceptions.ProtocolAdapterException;
+
+public class ProtocolServiceLookup {
```

Comment:
this name is too generic as is resides in the configuration package. Consider a renamed to ConfigurationServiceLookup.

Reply:
The fully qualified name of this class is `org.opensmartgridplatform.adapter.protocol.dlms.domain.commands.configuration.service.ProtocolServiceLookup`. I feel no need to repeat `ProtocolService` in the class name. We have packages for a reason.